src/Akeneo/Channel/back/API/Query/Channel.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Akeneo\Channel\API\Query;
+
+/**
+ * @copyright 2022 Akeneo SAS (https://www.akeneo.com)
+ * @license   https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+final class Channel
+{
+    /**
+     * @param string          $code
+     * @param array<string>   $localeCodes
+     * @param LabelCollection $labels
+     * @param array<string>   $activeCurrencies
```

Comment:
You can remove already type enforced by php
```suggestion
     * @param array<string>   $localeCodes
     * @param array<string>   $activeCurrencies
```